src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -226,11 +229,18 @@ def run_gridscan(
 
     @bpp.set_run_key_decorator("do_fgs")
     @bpp.run_decorator(md={"subplan_name": "do_fgs"})
-    @bpp.stage_decorator([fgs_composite.eiger])
     def do_fgs():
-        yield from bps.wait()  # Wait for all moves to complete
-        yield from bps.kickoff(fgs_motors)
-        yield from bps.complete(fgs_motors, wait=True)
+        try:
+            yield from bps.wait()  # Wait for all moves to complete
+            yield from bps.kickoff(fgs_motors)
+            yield from bps.complete(fgs_motors, wait=True)
+        finally:
+            yield from bps.unstage(fgs_composite.eiger)
```

Comment:
As discussed offline with @callumforrester it's a bit odd we're manually arming but disarming with `unstage`. We should address this in https://github.com/DiamondLightSource/dodal/issues/67

Reply:
Another thought might be to rename the arming group to something like `"deferred_stage"`, which makes the symmetry more obvious

Reply:
I like `arm`, it makes more sense to the scientists.